src/platform/packages/shared/kbn-esql-validation-autocomplete/src/autocomplete/commands/stats/index.ts

```diff
@@ -43,11 +43,16 @@ export async function suggest({
     await getColumnsByType('any', [], { advanceCursor: true, openSuggestions: true }),
     true
   );
-  const controlSuggestions = getControlSuggestionIfSupported(
-    Boolean(supportsControls),
-    ESQLVariableType.FUNCTIONS,
-    getVariables
-  );
+  const lastCharacterTyped = innerText[innerText.length - 1];
+
+  const controlSuggestions =
+    lastCharacterTyped !== '?'
+      ? getControlSuggestionIfSupported(
+          Boolean(supportsControls),
+          ESQLVariableType.FUNCTIONS,
+          getVariables
+        )
+      : [];
```

Comment:
This is another occurrence that should be covered by `suggestForExpression` once https://github.com/elastic/kibana/issues/216492 is finished.

Reply:
Actually this change is not needed (leftover from a previous implementation I did before release the PR). I removed it here https://github.com/elastic/kibana/pull/216839/commits/15df99142e3dd1e46e070b2a678a73fa87f1dff6